app/test_split_integer.py

```diff
@@ -2,20 +2,31 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 10
+    num_parts = 5
+    res = split_integer(value, num_parts)
+    assert sum(res) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    expected = split_integer(10, 5)
+    result = [2, 2, 2, 2, 2]
+    assert expected == result
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    expected = split_integer(2, 1)
+    result = [2]
+    assert expected == result
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    expected = split_integer(2, 3)
```

Comment:
Avoid creating unnecessary variables for simple tests. Directly assert the sorted result: `assert sorted(split_integer(2, 3)) == split_integer(2, 3)`.